src/components/SearchInput.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  ViewPropTypes,
+  StyleSheet,
+  Text,
+  View,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native'
+
+// Theme
+import theme from '../theme'
+
+const { colors } = theme
+
+const styles = StyleSheet.create({
+  searchContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    backgroundColor: colors.main.white.rgb,
+    borderRadius: 8,
+    marginVertical: 16,
+    marginHorizontal: 16,
+    height: 36,
+  },
+  search: {
+    flex: 1,
+    paddingLeft: 8,
+    backgroundColor: colors.main.lightGrey.rgb,
+    fontWeight: '400',
+    fontSize: 15,
+    lineHeight: '140%',
+    marginVertical: 10,
+  },
+})
+
+class SearchInput extends React.Component {
```

Comment:
Any chance this component can be build on top of the new `TextInput` ?

Reply:
I am working on that right now.